Add github issue templates (#107)
exasol/pyexasol: .github/ISSUE_TEMPLATE/bug.md

---
name: 🐞 Bug Report
about: File a bug report
title: 🐞 <Insert Title>
labels: bug
assignees: ''

---

### Checklist
In order to improve the time, how long a bugfix does take, please make sure you address as much as possible of the following checklist.

- [ ] I have reproduced the issue with at least the latest released version of \<PROJECT-NAME\>.
- [ ] I have added references to issues that sound similar.

> **Tip: Adding screenshots, outputs and logs help to improve clarity.**

Reviewer: ```suggestion
**Tip: Adding screenshots, outputs and logs might help to improve clarity.**
```

# Summary
<!---
Describe the bug: A clear and concise description of what the bug is.
-->

## Reproducing the Issue

> 

Reviewer: ```suggestion
```

Reproducibility: <pick one of> sporadic, regularly, always
<!---
Describe how to reproduce the bug, e.g.:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error
-->
### Steps to reproduce the behavior:
1. <step 1>
2. <step 2>
3. ....

## Expected Behaviour
<!---
Describe the expected behaviour: A clear and concise description of what you expected to happen.
-->

## Actual Behaviour
<!---

> Describe the actual observed behaviour: A clear and concise description of what you actually to happen.

Reviewer: ```suggestion
Describe the actual observed behaviour: A clear and concise description of what actually happened.
```

> Screenshot: If applicable, add screenshots to help explain your problem.

Reviewer: ```suggestion
Screenshot: If applicable, add screenshots to help explaining your problem.
```

-->

## Root Cause (optional)
<!---
If known
-->

## Context
<!---
Describe the actual observed behaviour: A clear and concise description of what you observed.

> Screenshot: If applicable, add screenshots to help explain your problem.

Reviewer: ```suggestion
Screenshot: If applicable, add screenshots to help explaining your problem.
```
